lifecycle_policy.md

```diff
@@ -0,0 +1,168 @@
+
+Project progression - movement from one stage to another - allows projects to participate at the level that is most appropriate for them given where they are in their lifecycle. Regardless of stage, all High Performance Software Foundation projects benefit from a deepened alignment with existing projects, and access to mentorship, support, and foundation resources.
+
+Capitalized terms not otherwise defined in this Project Lifecycle Policy have the meanings ascribed to them in the Charter of the High Performance Software Foundation.
+
+
+## II. Project Proposal Process
+
+### Introduction
+This governance policy sets forth the proposal process for projects to be accepted into the High Performance Software Foundation. The process is the same for both existing projects which seek to move into the High Performance Software Foundation and new projects to be formed within the High Performance Software Foundation.
+
+### Project Proposal Requirements
+Projects must be formally proposed via GitHub. Project proposals submitted to the High Performance Software Foundation should provide the following information to the best of their ability:
+
+* name of project
+* project description (what it does, why it is valuable, origin and history)
+* statement on alignment with the High Performance Software Foundation mission as expressed in the [HPSF Roles and Values](https://hpsf.io/#goals)
+* link to *current* Code of Conduct (if one is adopted already)
+* project license
+* source control (e.g., GitHub, Gitlab)
+* issue tracker (e.g., GitHub, Gitlab)
+* external dependencies (including licenses)
+* release methodology and mechanics
+* names of initial committers, if different from those submitting proposal
+* brief description of current leadership team and decision-making process
+* link to any documented governance practices
+* list of project's official communication channels (E.g., slack, irc, mailing lists)
+* link to project's website
+* links to social media accounts
+* brief description of current user community
+* brief description of current contributor community
+* existing financial sponsorship and funding institutions
+* existing legal entity, if any
+* sponsor from the TAC, if identified (a sponsor helps mentor projects)
+* preferred project lifecycle stage to enter HPSF (see stages below)
+* infrastructure needs or requests (there will be opportunities to change this after joining HPSF)
+
+### Project Acceptance Process
+* Projects are required to present their proposal at a TAC meeting
+* The TAC may ask for changes to bring the project into better alignment with the High Performance Software Foundation (adding a governance document to a repository or adopting a Code of Conduct, for example).
+* The project will need to make these changes in order to progress further.
+* Projects are accepted by a 2/3 majority of the TAC.
+* Vote majorities will be calculated as a percentage of the votes received
```

Comment:
I'm not changing any language in this initial PR so we can get the policy into the repo as approved by the TAC, but I think we should change this line to further clarify. Something like

> Vote majorities will be calculated as a percentage of votes received -- absences that do not affect quorum will change the required number of votes for project acceptance, but abstentions will count against the required 2/3 majority.